modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -93,22 +93,125 @@ image::odata-query-option-attributes.png["odataRequestAttributes is highlighted
+** endwith
+* $apply
+* $compute
+* $expand
+
+=== How Does Transform to SQL Select Work in Odata 4?
+
+The OData protocol defines a series of conventions that enable clients to query identified resources using URLs. The list of queryable entities, their fields, and the relationships between them are defined in a data model using simple HTTP messages.
+As OData shares some similarities with Java database connectivity (JDBC), the `transform-to-sql-select` operation takes advantage of such similarities to help the user create SQL queries for those resources. The resulting message of the operation is a prepared statement with the SQL `select` sentence as the payload along with the parameters name and values map as the attributes.
+
+=== How Do I Implement the Transform to SQL Select Operation Using Apikit for Odata 4?
+
+<apikit-odata:transform-to-sql-select> is the name of the XML element that represents this operation, which has the following parameters: 
+
+|====
+| *Parameter name* | *Description*
+| Display name | This parameter configures the operation name.
+| Module configuration | Name of the related-module configuration.
+| Entity set name | The name of the entity set that you are trying to fetch #[attributes.entitySetName].
+| Page size | A number that defines the amount of results per page during server-side-pagination.
+| Entity type name | A string obtained from the OData request using #[attributes.odataRequestAttributes], which is specified by the entity data model.
+| Entity type fields | An array of strings obtained from the OData request using #[attributes.odataRequestAttributes], which provides the list of property fields from the entity type.
+| Entity type keys | An object obtained from the OData request using #[attributes.odataRequestAttributes], which provides the map of entity key names and values.
+| System query options | Parameter obtained from the OData request using #[attributes.odataRequestAttributes].
+| Search transformation enabled | Parameter to define the behavior of the transformation.
+| Strict search term match | Parameter to define the behavior of the transformation.
+|====
+
+=== $filter Expressions in OData 4 
+
+​​The `$filter` expression in OData uses comparison, logical, arithmetic, and grouping operators. You can access the list operators with their corresponding meaning at http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations[the OData4 Specification^]. APIkit does not support `has` and `divby` operators.
+
+=== Expressions Supported by Transform to SQL Select
+
+OData supports a variety of expressions for filtering that you can find in the [specification^](http://docs.oasis-open.org/odata/odata/v4.01/cs01/part1-protocol/odata-v4.01-cs01-part1-protocol.html#sec_BuiltinFilterOperations). The `transform-to-sql-select` operation supports the following subset of unary and binary operators, along with the following method operators.
```

Comment:
is the link to the specification well formatted? It has a different format as the one in the previous section.

Reply:
Good catch! Resolved!